Sofa/framework/Config/CMakeLists.txt

```
@@ -130,6 +130,20 @@ set(SOFACONFIG_LINK_OPTIONS "")
 set(SOFACONFIG_LINK_OPTIONS_DEBUG "")
 set(SOFACONFIG_LINK_OPTIONS_RELEASE "")
 
+include(CheckIPOSupported)
+check_ipo_supported(RESULT IS_IPO_SUPPORTED OUTPUT IPO_ERROR)
+
+## Link-time optimization
+if (IS_IPO_SUPPORTED)
+    # Focus on max speed with link-time optimization
+    option(SOFA_ENABLE_LINK_TIME_OPTIMIZATION "Enable interprocedural optimization" OFF)
+    if (SOFA_ENABLE_LINK_TIME_OPTIMIZATION)
+        set(CMAKE_INTERPROCEDURAL_OPTIMIZATION TRUE)
```

```suggestion
        set(CMAKE_INTERPROCEDURAL_OPTIMIZATION_RELEASE TRUE)
```

I would set the IPO only for release mode
(thanks to https://github.com/ninja-build/ninja/blob/master/CMakeLists.txt)